### DISCUSSION TITLE
Feature Request: Add license file to include.zip

### DISCUSSION THREAD

**Discussion by falk-werner** [nlohmann/json#2487] [Ideas]
This is a feature request. I would like to ask to add the license file (LICENSE.MIT) to the include.zip release artifact.

#### What is the issue you have?

I am creating software for an embedded device using a [Yocto](https://www.yoctoproject.org/)-like build system. To a build an open source package (like nlohman/json) the build system fetches the sources, extract and build them and finally installs all the stuff needed to the system.

One use case for the embedded device is to present the licenses of all included open source packages to the user. Therefore the license file is needed.

Currently I use the released source archive, which contains all I need. But I would like to use the released include.zip archive instead, because of the reduced file size. 

#### Please describe the steps to reproduce the issue.

Download [include.zip](https://github.com/nlohmann/json/releases/download/v3.9.1/include.zip) form the download section.
A license file is not included.

#### Can you provide a small but working code example?

No, since this is no code issue.

#### What is the expected behavior?

The include.zip archive found in the download sections contains a license file (LICENSE.MIT) in the root directory of the archive.

#### And what is the actual behavior instead?

There is no license file included in include.zip archive.

#### Which compiler and operating system are you using?

This is not relevant to the issue.

#### Which version of the library did you use?

- [x ] other release - please state the version: 3.7.0

(I already noticed that the source archive is much smaller in the latest version 3.9.1)

#### If you experience a compilation error: can you [compile and run the unit tests]

This is not relevant to the issue.

**Comment by nlohmann** [comment]
Done in https://github.com/nlohmann/json/commit/d028eff9e1c8ac43be8eedda7126d83f7f8c08c0.
